index.js

```diff
@@ -1,33 +1,38 @@
-var async = require('async');
-
 var apple = require('./lib/apple');
 var google = require('./lib/google');
+var amazon = require('./lib/amazon');
 var windows = require('./lib/windows');
 var constants = require('./constants');
 
 module.exports.APPLE = constants.SERVICES.APPLE;
 module.exports.GOOGLE = constants.SERVICES.GOOGLE;
 module.exports.WINDOWS = constants.SERVICES.WINDOWS;
+module.exports.AMAZON  = constants.SERVICES.AMAZON;
 
-module.exports.config = function (configIn) {
-	apple.readConfig(configIn);
-	google.readConfig(configIn);
+module.exports.config = function (configIn, platform) {
```

Comment:
This change will change the usage of .config(). It would be better if we don't require platform.